android/src/main/java/com/horcrux/svg/ImageView.java

```diff
@@ -48,6 +51,8 @@ class ImageView extends RenderableView {
   private String mAlign;
   private int mMeetOrSlice;
   private final AtomicBoolean mLoading = new AtomicBoolean(false);
+  private final EventDispatcher mEventDispatcher =
```

Comment:
I would also call this method straight in the code that uses it, something like here: https://github.com/software-mansion/react-native-screens/blob/5cfafbbf42a3b6ebcdb7772012f760c64bb305df/android/src/main/java/com/swmansion/rnscreens/Screen.kt#L272

Reply:
good point, thanks.